content/software-security/learning-labs/ll202505.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Learning Lab May 2025 - Chainguard Libraries for Java"
```

Comment:
I think we can remove the date from this. In the case where there is more than one of the same content, we can choose one to bring over